desk/app/chat.hoon

```diff
@@ -855,15 +855,15 @@
       ?^  tim=(slaw %da i.t.t.path)  `u.tim
       `(slav %ud i.t.t.path)
     :^  ~  ~  %chat-heads
-    !>((old-heads-3 (heads since)))
+    !>(`chat-heads:old-3`(old-heads-3 (heads since)))
```

Comment:
Not that anything is wrong with this, but generally I have refrained from casting where the function is meant to enforce the type. Here, `+old-heads-3` is never going to change in its output type, that is `chat-heads:old-3`.